docs/_files/code/Visio_beispiel.py

```diff
@@ -0,0 +1,274 @@
+            label="Heat Demand",
+            inputs={
+                bus_heat: flows.Flow(fix=data["demand_el"], nominal_value=2)
+            },
+        )
+    )
+
+    # create source object representing the gas commodity
+    energysystem.add(
+        components.Source(
+            label="Gas Grid",
+            outputs={bus_gas: flows.Flow()},
+        )
+    )
+
+    # create fixed source object representing wind power plants
+    energysystem.add(
+        components.Source(
+            label="Power Grid",
+            outputs={bus_electricity: flows.Flow()},
+        )
+    )
+
+    # create fixed source object representing pv power plants
+    energysystem.add(
+        components.Source(
+            label="Pv Plant",
+            outputs={
+                bus_electricity: flows.Flow(
+                    fix=data["pv"], nominal_value=582000
+                )
+            },
+        )
+    )
+
+    # create simple sink object representing the electrical demand
+    # nominal_capacity is set to 1 because demand_el is not a normalised series
+    energysystem.add(
+        components.Sink(
+            label="Electricity Demand",
+            inputs={
+                bus_electricity: flows.Flow(
+                    fix=data["demand_el"], nominal_value=1
+                )
+            },
+        )
+    )
+
+    # create simple converter object representing a gas power plant
+    energysystem.add(
+        components.Converter(
+            label="Gas boiler",
+            inputs={bus_gas: flows.Flow()},
+            outputs={bus_heat: flows.Flow(variable_costs=50)},
+            conversion_factors={bus_electricity: 0.58},
+        )
+    )
+
+    energysystem.add(
+        components.Converter(
+            label="Heatpump",
+            inputs={bus_electricity: flows.Flow()},
+            outputs={bus_heat: flows.Flow(variable_costs=50)},
+            conversion_factors={bus_electricity: 0.58},
+        )
+    )
+
+    # create storage object representing a battery
+    nominal_capacity = 10077997
+    nominal_capacity = nominal_capacity / 6
+
+    # battery_storage = components.GenericStorage(
+    #     label=STORAGE_LABEL,
+    #     inputs={bus_electricity: flows.Flow()},
+    #     outputs={bus_electricity: flows.Flow(variable_costs=0.001)},
+    #     loss_rate=0.00,
+    #     initial_storage_level=None,
+    #     inflow_conversion_factor=1,
+    #     outflow_conversion_factor=0.8,
+    # )
+
+    # energysystem.add(battery_storage)
+
+    ##########################################################################
+    # Optimise the energy system and plot the results
+    ##########################################################################
+
+    logging.info("Optimise the energy system")
+
+    # initialise the operational model
+    energysystem_model = Model(energysystem)
+
+    esgr = ESGraphRenderer(
+        energysystem_model,
+        legend=False,
+        filepath="D:/ES.svg",
```

Comment:
```suggestion
        filepath="ES.svg",
```

The example with `D:/` will only work for windows users and for windows users having a drive under this address.